integtests/IntegTests/HBVersion.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace IntegTests
+{
+    class HBVersion
```

Comment:
```suggestion
    public class HBVersion
```
Is it working ? I have errors when launching version tests

Reply:
Not on my machine. I'm on it to see what happens.